pyproject.toml

```diff
@@ -320,7 +321,6 @@ lint.select = [
     "DTZ",  # flake8-datetimez
     "T10",  # flake8-debugger
     "EXE",  # flake8-executable
-    "ISC",  # flake8-implicit-str-concat
```

Comment:
FTR, why was this removed? I could not find it mentioned in the description

Reply:
This does not seem to conflict with any other `ruff` settings and I'd say it's still useful to prevent unnecessary string concatenation. Just verified that `ruff format` itself would keep it unnoticed. Let's put it back.

Reply:
Done in bb027a13.